### DISCUSSION TITLE
Noob here. Anyone else running in TrueNAS as an app? Can't seem to find the conf.yaml file

### DISCUSSION THREAD

**Discussion by jbo118** [Lissy93/dashy#1214] [Help and Support]
Hi all, first time poster. I've loved setting up Dashy so far but the config editor in the web UI is having issues for me, seems to lose the changes I make every so often. Yes, I know to save to disk, not locally. I'd like to edit the yaml directly, but can't seem to find it within my truenas files. I changed the setup from PVC to host path hoping that when I looked into the folder I designated as the host path directory, I'd find the conf.yml in there (the parent folder is an SMB share) and be able to edit it there. But the host directory is empty when I check. I also tried reinstalling with the storage type host path, instead of changing it after the fact. No success. Any other TrueNAS users out there able to weigh in?

**Comment by lissy93** [comment]
I'm not too sure on TrueNAS specifically, so hopefully someone else can weigh in with that. But the config file is in `public/conf.yml`, so if it's Docker it'd be `/app/public/conf.yml`.

In my setup, I just pass through the file to the container and modify it in VS Code web, that way it's always backed up even if the container is deleted.

**Reply by jbo118** [reply]
Thanks for the response! I know the file path it's SUPPOSED to be on, but just can't manage to reconcile that with the way TrueNAS containerizes the app. I'd love to do VS after, but can't even get there at this point.

**Reply by cygnusnsf** [reply]
mine was in TrueNAS location = /mnt/.ix-apps/app_mounts/dashy/data
